src/core/TrainController.hpp

```diff
@@ -71,50 +72,53 @@ namespace godot {
 
             Dictionary get_config() const;
             void update_config(const Dictionary &p_config);
-            void set_config_property(String &key, Variant &p_value);
+            static void set_config_property(String &key, Variant &p_value);
```

Comment:
???

Reply:
It was possible to make it static without crashing anything, so I did that